src/components/study-container.js

```diff
@@ -502,7 +501,6 @@ export function StudyContainer({ view, onChangeTab }) {
                         (node) => node.id === firstSelectedNode.id
                     ),
                 };
-                currentNodeRef.current = ModelFirstSelectedNode;
```

Comment:
@jonenst @sBouzols @Meklo ok with that ?
currentNodeRef.current is initialized at row 634.
Thus, at study opening, I will have previousCurrentNode=undefined. And the first and only execution of the network load useEffect will work fine.